src/frontend/src/components/SearchList.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import {
+    selectNeighborhoodFilters,
+    selectNeighborhoodFilterSuggestions,
+    setNeighborhoodFilters,
+} from "reducers/neighborhoods/neighborhoodsSlice";
+import { getRankedNeighborhoodLists } from "reducers/neighborhoods/neighborhoodsThunk";
+import type { FiltersState } from "reducers/neighborhoods/types";
+import { useAppDispatch, useAppSelector } from "store/store";
+
+import {
+    Autocomplete,
+    type Suggestion,
+} from "./base/Autocomplete/Autocomplete";
+
+const SearchList = ({ isMobile = false }) => {
+    const { t } = useTranslation();
+    const dispatch = useAppDispatch();
+    const neighborhoodSuggestions = useAppSelector(
+        selectNeighborhoodFilterSuggestions
+    );
+    const filters = useAppSelector(selectNeighborhoodFilters);
+    const [filtersBuffer, setFiltersBuffer] = useState<FiltersState | null>(
```

Comment:
Similar to the suggestion in the other PR- we could try initializing the filterBuffer by `{ ...filters }`. Though I'd suggest keeping the subsequent useEffect. What do you think?

Similar applies to SearchList.